src/client/scripts/InitHome.ts

```diff
@@ -10,8 +10,141 @@
 module StopByStop {
 
     export class InitHome {
+ 
+        //The functionality below pulls the image url based on lat long
+        //And populates the images inside a div
+        //If an image is not available, it is shown as a blank div
+        public static addImagesDynamically(prevPlace,currentLocationString) {
+            //$('#to').on('click', function (event) {
+            if (AppState.current.app != SBSApp.Web) {    
+                var place = $('#from').data('place');
+                
+                if(place===null){
+                    return true;
+                }
+                
+                if(prevPlace!=null && prevPlace.n===place.n){
+                    
+                    // Dont proceed further as the same place is being selected
+                    return true;
+                }
+
+                // Continue with the processing if the previous place doesnt match with the current returned place
+                prevPlace =  place;
+                //Remove any div contents from the previous population before adding new divs
+                $("#Images").empty();
+
+                var placesNearbyUrl = "";
+                
+                //If the place returned is the current location, the processing should be different, 
+                //should be picked up from place.i
+                if(place.n===currentLocationString)
+                {
+                    var modifiedCurrentLocation = place.i.replace(",", "/");
+                    placesNearbyUrl= 'https://www.stopbystop.com/placesnearby/'+modifiedCurrentLocation;
+                }
+                else
+                {
+                    placesNearbyUrl= 'https://www.stopbystop.com/placesnearby/'+place.l.a+'/'+place.l.o;
+                }
```

Comment:
Please include placesnearby to AppState.current.urls (just like AppState.current.urls.RouteUrl)